doc/api/vm.markdown

```diff
@@ -36,6 +36,12 @@ The options when creating a script are:
   code are controlled by the options to the script's methods.
 - `timeout`: a number of milliseconds to execute `code` before terminating
   execution. If execution is terminated, an [`Error`][] will be thrown.
+- `cachedData`: an optional `Buffer` with V8's code cache data for the supplied
+  source. When supplied `cachedDataRejected` value will be set to either
+  `true` or `false` depending on acceptance of the data by V8.
```

Comment:
> allow compiling with reduced set of CPU features and using it on CPU that actually have wider feature support.

This would leave performance on the table. You would want to use all the CPU features available on the machine that the code is actually running on. Over and above the CPU feature set (the architecture), modern JIT compilers tune code to the exact micro-architecture the code is going to be running on. For example, the IBM Power 5 and IBM Power 6 were not very different on the surface from CPU feature set point of view, but are radically different micro-architectures. Code compiled with the Power 5 feature set works fine on Power 6 but you can get significant performance improvements by tweaking the instruction selection and scheduling if you knew the code is going to run on a Power 6.

Shipping code built in the lab will leave these performance opportunities on the table.